docs/getting_started/index.rst

```diff
@@ -1,115 +1,164 @@
 Getting Started
 ===============
 
-.. note:: This document assumes you have a `working Ansible
+.. Note:: This document assumes you have a `working Ansible
           installation`_. If you don't, install Ansible before
           continuing. This can be done simply by running ``pip install -r
           requirements.txt`` from the root of the project.
 
-The Mantl project uses Ansible to bring up nodes and clusters. This generally
-means that you need three things:
+          It also assumes you have a working Terraform installation which you
+          can download from `Terraform downloads`_.
 
- 1. hosts to use as the base for your cluster
- 2. an `inventory file`_ with the hosts you want to be modified.
- 3. a playbook to show which components should go where.
+General Information about Mantl with Ansible
+============================================
 
-Provisioning Cloud Hosts
-------------------------
+The Mantl project uses Ansible to bring up
+nodes and clusters. This generally means that you need three things:
+
+1. Hosts to use as the base for your cluster
+2. An `inventory file`_ with the hosts you want to be modified. Mantl includes
+   an Ansible inventory for terraformed hosts, `terraform.py`_.
+3. A playbook matching Ansible roles to hosts. Mantl organizes its components
+   in `sample.yml`_, which we recommend copying to ``mantl.yml`` for the
+   possibility of later customization. You can read more about `playbooks`_ in
+   the `Ansible docs`_.
+
+Preparing to provision Cloud Hosts
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
 The playbooks and roles in this project will work on whatever provider
 (or metal) you care to spin up, as long as it can run CentOS 7 or
-equivalent. However, here are some guides to get started on common
-platforms:
+equivalent.
 
-.. toctree::
-   :maxdepth: 1
+Your hosts will have to be accessible with your SSH key. If you're unfamiliar
+with SSH keys, please read `this blog post
+<https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys--2>`_.
```

Comment:
ix-nay on the inks-lay that say "this `x`". Try "DigitalOcean's guide to setting up SSH keys" instead. That way everyone knows what they're getting themselves into by clicking the link.